### ISSUE TITLE
[Harbor] trivy scanner is not working

### ISSUE THREAD

**Issue by HSoulat** [bitnami/vms#935]
### Platform

Azure

### bndiagnostic ID [know more about bndiagnostic ID](https://docs.bitnami.com/general/how-to/understand-bndiagnostic/)

not uploaded

### bndiagnostic was not useful. Could you please tell us why?

not connectivity related

### Describe your issue as much as you can

On latest Azure Harbor bitnami image, any trivy scan fail with the following traceback. It seems that the options passed in trivy's call are depreciated.
```

--------- Logs of report 8d84e20c-e85a-4752-b00a-9700632113bf ----------
2023-05-24T06:18:01Z [DEBUG] [/pkg/scan/job.go:376]: registration:
2023-05-24T06:18:01Z [INFO] [/pkg/scan/job.go:387]: {
...
}
2023-05-24T06:18:01Z [DEBUG] [/pkg/scan/job.go:376]: scanRequest:
2023-05-24T06:18:01Z [INFO] [/pkg/scan/job.go:387]: {
...
}
2023-05-24T06:18:01Z [INFO] [/pkg/scan/job.go:167]: Report mime types: [application/vnd.security.vulnerability.report; version=1.1]
2023-05-24T06:18:01Z [INFO] [/pkg/scan/job.go:224]: Get report for mime type: application/vnd.security.vulnerability.report; version=1.1
2023-05-24T06:18:03Z [DEBUG] [/pkg/scan/job.go:237]: check scan report for mime application/vnd.security.vulnerability.report; version=1.1 at 2023/05/24 06:18:03
2023-05-24T06:18:03Z [ERROR] [/pkg/scan/job.go:294]: check scan report with mime type application/vnd.security.vulnerability.report; version=1.1: running trivy wrapper: running trivy: exit status 1: 2023-05-24T08:18:02.447+0200	[33mWARN[0m	'--security-checks' is deprecated. Use '--scanners' instead.
2023-05-24T08:18:02.453+0200	[31mFATAL[0m	init error: WASM module error: module load error: module walk error: lstat /root/.trivy/modules: permission denied
: general response handler: unexpected status code: 500, expected: 200
```

Is there any workaround ? patch planned ?

**Comment by mdhont** [comment]
The deprecated flag message is a warning, so that's not the issue. Could you tell me the exact steps to reproduce the issue?

**Comment by HSoulat** [comment]
I've only deployed the last available image on Azure and pulled an image from docker hub.

**Comment by mdhont** [comment]
I've been able to reproduce the issue and I will create a task for fixing it. Thanks for reporting, I will update this thread with further information.

**Comment by HSoulat** [comment]
Great, 

When the fix will be available, is there a way to update / upgrade bitnami images or application installed on it ? or do we need to replace the VM at each patch ?

**Comment by mdhont** [comment]
You would need to launch a new instance with the new version and migrate the data there.

**Comment by HSoulat** [comment]
still relevant

**Comment by joancafom** [comment]
Hi @mpostelnicu @HSoulat 

I am glad to announce that the issue was fixed in the new release of our Harbor VM, `2.10.0-1-r01`. It is already available in both Azure and AWS (and it is making its way to other platforms as well) 🚀 .
